packages/mask/src/extension/popups/pages/Wallet/components/AssetsList/index.tsx

```diff
@@ -1,40 +1,54 @@
+import { Icons } from '@masknet/icons'
+import { FormattedBalance, FormattedCurrency, ImageIcon, TokenIcon } from '@masknet/shared'
+import { NetworkPluginID, PopupRoutes } from '@masknet/shared-base'
+import { makeStyles } from '@masknet/theme'
+import { useNetworkDescriptors } from '@masknet/web3-hooks-base'
+import { formatBalance, formatCurrency, isGte, isLessThan, type FungibleAsset } from '@masknet/web3-shared-base'
+import { isNativeTokenAddress, type ChainId, type SchemaType } from '@masknet/web3-shared-evm'
+import { Box, List, ListItem, ListItemIcon, ListItemText, Typography, type TypographyProps } from '@mui/material'
+import { isNaN } from 'lodash-es'
 import { memo, useCallback, useState } from 'react'
 import { useNavigate } from 'react-router-dom'
 import { useContainer } from 'unstated-next'
-import { type ChainId, isNativeTokenAddress, type SchemaType } from '@masknet/web3-shared-evm'
-import { PopupRoutes } from '@masknet/shared-base'
-import { List, ListItem, ListItemText, Typography } from '@mui/material'
-import { makeStyles } from '@masknet/theme'
-import { Icons } from '@masknet/icons'
-import { TokenIcon, FormattedBalance } from '@masknet/shared'
-import { WalletContext } from '../../hooks/useWalletContext.js'
-import { isNaN } from 'lodash-es'
-import { formatBalance, type FungibleAsset, isGreaterThanOrEqualTo, isLessThan } from '@masknet/web3-shared-base'
 import { useI18N } from '../../../../../../utils/index.js'
+import { WalletContext } from '../../hooks/useWalletContext.js'
 
 const useStyles = makeStyles()((theme) => ({
     list: {
-        backgroundColor: '#ffffff',
-        padding: 0,
+        backgroundColor: theme.palette.maskColor.bottom,
+        padding: theme.spacing(2),
     },
     item: {
         padding: 14,
-        borderBottom: '1px solid #F7F9FA',
         cursor: 'pointer',
+        marginBottom: theme.spacing(1),
+        borderRadius: 8,
+        '&:hover': {
+            backgroundColor: theme.palette.maskColor.bg,
+        },
     },
     tokenIcon: {
-        width: 20,
-        height: 20,
+        width: 36,
+        height: 36,
     },
-    text: {
-        margin: '0 0 0 15px',
+    badgeIcon: {
+        position: 'absolute',
+        right: -6,
+        bottom: -4,
+        border: `1px solid ${theme.palette.common.white}`,
+        borderRadius: '50%',
     },
-    symbol: {
-        color: '#1C68F3',
-        fontSize: 14,
+    text: {
+        marginLeft: 14,
+        fontSize: 16,
+        fontFamily: 'Helvetica',
+        fontWeight: 700,
+        color: theme.palette.maskColor.main,
     },
-    balance: {
-        fontSize: 14,
+    value: {
+        fontSize: 16,
+        fontFamily: 'Helvetica',
```

Comment:
Maybe we need to set the font family to global.

Reply:
Yep.